src/sync/sync_config.hpp

```diff
@@ -108,14 +108,15 @@ struct SyncConfig {
     util::Optional<std::array<char, 64>> realm_encryption_key;
     bool client_validate_ssl = true;
     util::Optional<std::string> ssl_trust_certificate_path;
+    std::function<sync::Session::SSLVerifyCallback> ssl_verify_callback;
```

Comment:
What's the definition of `SSLVerifyCallback`? If that's a function type (as the name implies), wrapping it in `std::function` seems unnecessary.

Reply:
Never mind. It's probably a typedef for an actual function type, which does need to be wrapped in a std::function.